content/en/platform/corda/5.0/key-concepts/cordapp-dev/intro/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Introducing CorDapps"
+date: 2023-07-25
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-cordapp-dev-intro
+    parent: corda5-key-concepts-cordapp-dev
+    weight: 1000
+section_menu: corda5
+---
+
+# Introducing CorDapps
+
+## Overview
+
+A CorDapp, or Corda distributed application, is distributed because parts of the application can be, or may need to be, executed on a Corda instance that is operated by another party that is a member of the same Corda application network.
+This is not the same as an application that has multiple instances, hosted on different environments.
+
+Consider the centralized application below. This is a single application that always executes within a single execution environment, regardless of the number of deployments.
+
+{{< 
+  figure
+	 src="centralized-application.png"
+   width="50%"
+	 figcaption="Centralized Application"
+>}}
+
+Similarly, centralized applications often use a services architecture that means different parts of the applications interact with each other. This could be inside an entity’s network or different parts may be operated by different parties. 
+
+{{< 
+  figure
+	 src="services-architecture.png"
+   width="50%"
+	 figcaption="Services Architecture"
+>}}
+
+However, distributed applications in Corda, while being deployed by different parties in the network, are unique in that parts of the application execute in the different environments as part of the same execution context.
+
+{{< 
+  figure
+	 src="distributed-application.png"
+   width="50%"
+	 figcaption="Distributed Application"
+>}}
+
+## Workflows, Contracts, and States
+
+CorDapps are usually composed of workflows, contracts, and states. This section illustrates that contracts and states are optional, but a CorDapp must always define at least one workflow, often referred to simply as a flow.
+
+Flows support the [orchestration layer]({{< relref "../../fundamentals/CorDapps/_index.md#orchestration-layer--flows" >}}), and are written in a JVM compatible language and are hosted by Corda. Typically, flows create or transfer states on behalf of a member of the application network (a virtual node) who seeks to form [consensus]({{< relref "../../fundamentals/CorDapps/_index.md#consensus-layer" >}}) with their peer nodes with the help of the notary.
```

Comment:
Is this correct with double dashes (--):
```
/_index.md#orchestration-layer--flows
```

Reply:
Yes - there's a dash in the title.